src/popup/lib/utils.ts

```diff
@@ -62,3 +62,7 @@ export function formatNumber(
     } else return value.toExponential()
   }
 }
+
+export function toWalletAddressUrl(s: string): string {
+  return s.startsWith('$') ? s.replace(/^\$/, 'https://') : s
```

Comment:
As it'll only replace first one and we tested it starts with `$`
```suggestion
  return s.startsWith('$') ? s.replace('$', 'https://') : s
```